environs/config/config.go

```diff
@@ -388,6 +391,7 @@ var defaultConfigValues = map[string]interface{}{
 	EgressSubnets:              "",
 	FanConfig:                  "",
 	CloudInitUserDataKey:       "",
+	BackupDirKey:               os.TempDir(),
```

Comment:
This isn't quite right - this will get the tmp dir on the client machine, not the controller
We want the default to be "" so we pass that as an arg to os.TempDir()

Reply:
Updated.